src/include/miopen/cat/problem_description.hpp

```diff
@@ -124,13 +120,13 @@ struct ProblemDescription : ProblemDescriptionBase
         {
             if(!xDescs[i]->IsPacked())
             {
-                MIOPEN_THROW(miopenStatusBadParm, "CatForward: Unpacked tensors not supported.");
+                return false;
             }
         }
 
         if(!yDesc.IsPacked())
         {
-            MIOPEN_THROW(miopenStatusBadParm, "CatForward: Unpacked tensors not supported.");
+            return false;
```

Comment:
Ditto

Reply:
changed it.

Reply:
[resolved here - see next review]

Reply:
remove exception, It returns false if it is not packed.